### PR TITLE
DT-1483 Fix tests due to new offender record

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java [hunk 1]
```diff
@@ -398,9 +398,7 @@ public void testFindOffendersWithDateOfBirthRange() {
 
         final var offenders = findOffenders(query);
 
-        assertThat(offenders).hasSize(9);
-        assertThat(offenders).extracting(PrisonerDetail::getOffenderNo)
-                .contains("A1234AA", "A1234AF", "A1234AL", "Z0019ZZ", "Z0020ZZ", "Z0021ZZ", "Z0022ZZ", "Z0023ZZ", "A1180MA");
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java:403]: this line was fine, it was using `contains` so extra items would be allowed

File: src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java [hunk 2]
```diff
@@ -600,9 +598,7 @@ public void testFindOffenderAliasesWithDateOfBirthRange() {
 
         final var offenders = findOffendersWithAliases(query);
 
-        assertThat(offenders).hasSize(9);
-        assertThat(offenders).extracting(PrisonerDetail::getOffenderNo)
-                .contains("A1234AA", "A1234AF", "A1234AL", "Z0019ZZ", "Z0020ZZ", "Z0021ZZ", "Z0022ZZ", "Z0023ZZ", "A1180MA");
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java:605]: this line was fine, it was using `contains` so extra items would be allowed

File: src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java [hunk 3]
```diff
@@ -265,7 +265,7 @@ public void testFindOffendersWithLocationFilterALL() {
 
         final var offenders = findOffendersWithAliasesFullResults(query);
 
-        assertThat(offenders).hasSize(53);
+        assertThat(offenders.size()).isGreaterThanOrEqualTo(53);
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java:268]: there is a `hasSizeGreaterThanOrEqualTo` method :grin:

**Review by petergphillips — APPROVED** [review]

File: src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java (hunk 1 above)
Comment (Mjwillis) [reply src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java:403]: duh

Comment (Mjwillis) [reply src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java:403]: duh - updated

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**